src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -545,6 +554,73 @@ private ImmutableArray<AdditionalTextFile> ResolveAdditionalFilesFromArguments(L
             return builder.ToImmutableArray();
         }
 
+        private static void ReportAnalyzerExecutionTime(TextWriter consoleOutput, AnalyzerDriver analyzerDriver, CultureInfo culture, bool isConcurrentBuild)
+        {
+            Debug.Assert(analyzerDriver.AnalyzerExecutionTimes != null);
+            if (analyzerDriver.AnalyzerExecutionTimes.IsEmpty)
+            {
+                return;
+            }
+
+            var totalAnalyzerExecutionTime = analyzerDriver.AnalyzerExecutionTimes.Sum(kvp => kvp.Value.TotalSeconds);
```

Comment:
Call me picky, but sum the TimeSpans and then do a single ".TotalSeconds" at the end.
i.e.
```C#
var totalAnalyzerExecutionTime = analyzerDriver.AnalyzerExecutionTimes.Sum(kvp => kvp.Value).TotalSeconds;

The reason is the math for .TotalSeconds involves a floating point multiply, whereas TImeSpan addition is integer math.